RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -447,6 +452,20 @@ void bedtime_pulls()
 		return;
 	}
 	
+	if(get_property("auto_bedtime_pulls_min_desirability").to_float() <= 5.0)
+	{
+		if(item_amount($item[potato alarm clock]) == 0 && storage_amount($item[potato alarm clock]) > 0)
+		{
+			pullXWhenHaveY($item[potato alarm clock], 1, 0);
+		}
+
+		if(my_daycount() == 1 && my_level() <= 8)
+		{
+			//this run looks like it will take a couple more days, give priority to good rollover equipment before level 11 quest pulls
+			bedtime_pulls_rollover_equip(5.0);
+		}
+	}
```

Comment:
```suggestion
	pullXWhenHaveY($item[potato alarm clock], 1, 0);
	float desirability = max(5.0, get_property("auto_bedtime_pulls_min_desirability").to_float());
	if(my_daycount() == 1 && my_level() <= 8)
	{
		//this run looks like it will take a couple more days, give priority to good rollover equipment before level 11 quest pulls
		bedtime_pulls_rollover_equip(desirability);
	}
```

Reply:
1. 
```
if(item_amount($item[potato alarm clock]) == 0 && storage_amount($item[potato alarm clock]) > 0)
{
	pullXWhenHaveY($item[potato alarm clock], 1, 0);
}
```
the if statement is wholly redundant with `pullXWhenHaveY`
as `pullXWhenHaveY(it, 1, 0)` already does this if statement

2. I don't think the potato clock should depend on
`if(get_property("auto_bedtime_pulls_min_desirability").to_float() <= 5.0)` but come before it.

3. if someone set desirability to higher than 5.0 than run it with their preferred number instead of disabling it. it will still raise lower values to 5.0 as the original code you wrote did

Reply:
rest of the PR looks good

Reply:
isn't potato alarm clock exactly = 5 desirability? actually I should place it after the rollover equipment improvements. >= 5 equipment could be counterclockwise or Crimbolex watch but clock will never be more than 5

Reply:
that is fair enough. updated suggestion to put it after it the desirability 5 bedtime pulls.